charts/opensearch-cluster/README.md

```diff
@@ -0,0 +1,24 @@
+# OpenSearch-k8s-operator
+
+The Kubernetes [OpenSearch Operator](https://github.com/Opster/opensearch-k8s-operator) is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards.
+
+## Getting started
+
+The Operator can be easily installed using helm on any CNCF-certified Kubernetes cluster. Please refer to the [User Guide](https://github.com/Opster/opensearch-k8s-operator/blob/main/docs/userguide/main.md) for more information.
+Once the operator is installed, opensearch cluster can be installed using helm as well in the same CNCF-certified Kubernetes cluster.
```

Comment:
Let give that a title so user will notice, something like :
`Install OpenSeachCluster Using Helm`